Source/Tests/LinuxSyscalls/x32/Socket.cpp

```diff
@@ -200,8 +205,8 @@ namespace FEX::HLE::x32 {
           HostHeader.msg_iov = &Host_iovec.at(0);
           HostHeader.msg_iovlen = guest_msg->msg_iovlen;
 
-          HostHeader.msg_control = alloca(guest_msg->msg_controllen);
-          HostHeader.msg_controllen = guest_msg->msg_controllen;
+          HostHeader.msg_control = alloca(guest_msg->msg_controllen+8);
```

Comment:
As a side comment, this needs to be *2 for crypt of the necrodancer to work. +8 causes it to infinite loop.